graph_api/relationship/relationship_model.py

```diff
@@ -1,20 +1,21 @@
-from typing import Set, Optional, Any
+from typing import Set, Optional, Any, List
 from pydantic import BaseModel
+from property.property_model import PropertyIn
 
 
 class RelationshipIn(BaseModel):
     """
         Model of relationship to acquire from client
 
         Attributes:
-            start_node (int): Id of node which starts connection
-            end_node (int): Id of node which ends connection
-            name (str): Name of the relationship
+            start_node (Optional[int]): Id of node which starts connection
+            end_node (Optional[int]): Id of node which ends connection
+            name (Optional[str]): Name of the relationship
     """
 
-    start_node: int
-    end_node: int
-    name: str
+    start_node: Optional[int]
+    end_node: Optional[int]
+    name: Optional[str]
```

Comment:
I'm not sure if this should be Optional

Reply:
I also cannot think of a case where there is a relationship without a start or end node. Perhaps there can be a relationship without a name.... @terzawad?

Reply:
For now, lets remove Optional from `start_node` and `end_node` and leave it for `name`.